dissect/target/volumes/md.py

```diff
@@ -1,7 +1,8 @@
 from collections import defaultdict
 from typing import BinaryIO, Iterator, Union
 
-from dissect.volume.md.md import MD, Device, find_super_block
+from dissect.volume.md import MD, MDPhysicalDisk
+from dissect.volume.md.md import find_super_block
```

Comment:
```suggestion
from dissect.volume.md.md import MD, MDPhysicalDisk, find_super_block
```
This would also work right?